### ISSUE TITLE
missing module in lelesk

### ISSUE THREAD

**Issue by arademaker** [letuananh/intsem.fx#6]
During the installation from source

```
$ ./setup.py
/home/user/.local/lib/python3.6/site-packages/delphin/interfaces/ace.py:37: FutureWarning: Responses from the ACE interface will use a response class (like delphin.interfaces.rest) in future releases and may not be entirely compatible with the current format.
  FutureWarning
Traceback (most recent call last):
  File "./setup.py", line 39, in <module>
    from coolisf import __author__, __email__
  File "/home/user/workspace/intsem.fx/coolisf/__init__.py", line 49, in <module>
    from .ghub import GrammarHub
  File "/home/user/workspace/intsem.fx/coolisf/ghub.py", line 47, in <module>
    from coolisf.config import read_config
  File "/home/user/workspace/intsem.fx/coolisf/config.py", line 46, in <module>
    from coolisf.common import write_file
  File "/home/user/workspace/intsem.fx/coolisf/common.py", line 43, in <module>
    from lelesk.util import ptpos_to_wn
ModuleNotFoundError: No module named 'lelesk.util'
```

**Comment by arademaker** [comment]
Same error if using the last release.

**Comment by letuananh** [comment]
This issue is finally fixed with coolisf version 0.2.3 release.

https://pypi.org/project/coolisf/
